.circleci/config.yml

```diff
@@ -276,6 +280,7 @@ jobs:
             # pin above.
             go get gorm.io/driver/mysql@v1.2.3
             go get gorm.io/driver/postgres@v1.2.3
+            go get gorm.io/driver/sqlserver@v1.3.0
```

Comment:
Looks like while the go.mod file for `gorm.io/driver/sqlserver@v1.3.0` says it's fine with the older gorm version, it's not. We probably need to go back to `v1.2.1`